lab-jeff/lib/server.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
Generally, you want your server files to be located in your project root rather than in the lib directory